pyproject.toml

```diff
@@ -5,7 +5,10 @@ build-backend = "flit_core.buildapi"
 [project]
 name = "aiida_orca"
 dynamic = ["version", "description"]
-authors = [{name = "Pezhman Zarabadi-Poor", email = "pzarabadip@gmail.com"}]
+authors = [
+    {name = "Pezhman Zarabadi-Poor", email = "pzarabadip@gmail.com"},
+    {name = "Daniel Hollas", email = "danekhollas@gmail.com"}
```

Comment:
Thanks @pzarabadip :heart_eyes:

Reply:
Cheers @danielhollas. Thank you very much.
I have setup a `publish-to-pypi` actions which is failing. Once it is fixed, it will be on `pypi` with less pain :)